app/javascript/stylesheets/palette.scss

```diff
@@ -1,9 +1,12 @@
 $alabaster: #f2f1eb;
 $silver-sand: #c0c0bf; // "30% black" in Stanford brand identity
 $stanford-cardinal: #8c1515;
+$stanford-cardinal-light: #B83A4B;
+$stanford-spirited-light: #F4795B;
 $stanford-cool-grey: #53565a; // used in SDR header (not yet added)
 $stanford-digital-blue: #006cb8;
 $stanford-digital-green: #008566;
 $stanford-digital-red: #b1040e;
+$stanford-digital-pink: #F3E7E7;
```

Comment:
This is TOTALLY a nitpick, but the other colors in `palette.scss` use lowercase hex digits and the new entry uses uppercase.

Reply:
I wonder if we want a different name, too, so as not to suggest that this is an official Stanford identity color. When googling that hex code, I see that some paint companies sell this as `Laughing Maiden` or `Ibis White`.

How about `$ibis-white: #f3e7e7; // light pinkish hue used in what's changing box` here?